include/d/d_mesg.h

```diff
@@ -47,7 +47,7 @@ class dMesg_tControl {
     void setRubyFont(JUTFont*) {}
     
     dMesg_tControl();
-    void do_word(u32);
+    const char* do_word(u32);
```

Comment:
also have to update the d_mesg.cpp file when changing return values to fix the build error

Reply:
Why didn't that show up locally?

Reply:
d_demo isn't a fully matched TU, so if you run `ninja`, it just compiles and links the fully matching TU, and makes sure they still match. run `ninja all_source` to compile all TUs and make sure none of them have errors (but `ninja all_source` doesn't check if the linked TUs actually match, so you have to run both to be sure there are no issues.)